packages/python-aiohttp/python-aiohttp.spec

```diff
@@ -5,8 +5,8 @@
 %global pypi_name aiohttp
 
 Name:           %{?scl_prefix}python-%{pypi_name}
-Version:        3.7.4
-Release:        4%{?dist}
+Version:        3.7.4.post0
```

Comment:
please drop this bump, it's badly versioned and there are no user-relevant changes in this silly release (I hate python versioning so much)

Reply:
Ack